reference/ToDo/src/ToDo/Views/TaskListPage.xaml

```diff
@@ -161,7 +161,14 @@
 					<VisualState.StateTriggers>
 						<AdaptiveTrigger MinWindowWidth="0" />
 					</VisualState.StateTriggers>
-				</VisualState>
+          <VisualState.Setters>
+            <Setter Target="TaskListColumn.Width" Value="*" />
+            <Setter Target="TaskListColumn.MinWidth" Value="300" />
+            <Setter Target="TaskColumn.Width" Value="2*" />
+            <Setter Target="TaskDetailsGrid.Visibility" Value="Collapsed" />
+            <Setter Target="TaskDetails.(uen:Region.Attached)" Value="false" />
+          </VisualState.Setters>
+        </VisualState>
```

Comment:
fix indentation to aligned with the StateTriggers node

Reply:
hmm the indentation looks fine to me, maybe just github shenanigans?

i see this in Changes:
![image](https://github.com/unoplatform/Uno.Samples/assets/70542961/e8b23f22-2336-4080-973a-20ba25f0be83)

Reply:
Ah, tabs versus spaces :) 
![image](https://github.com/unoplatform/Uno.Samples/assets/4793020/be1b1ed8-9b60-4456-8231-7a3a22ae3465)


Let's keep it all tabs

Reply:
I didn't realize at first but these setters are actually messing up the view on Mobile, for Android at least.

Reply:
I thought it could be because of the AutoLayout._____Alignment="Stretch", because from having worked on the other singleproject migrations, removing those would normally fix these weird crashes on Windows.

But here it didn't work, I ended up just adding this MinWidth instead (it looks even better on Android now):
https://github.com/unoplatform/Uno.Samples/blob/723ed8c3b7f9c8a6242585f55a035642079121ad/reference/ToDo/src/ToDo/Views/TaskListPage.xaml#L161

Reply:
Is it also messed up on `master`? If so we should leave it as is and fix in a later PR

Reply:
https://github.com/unoplatform/Uno.Samples/issues/760